infrastructure/templates/public-api/api-infrastructure-pipeline.yml

```diff
@@ -58,21 +66,29 @@ stages:
       displayName: 'List source files'
       inputs:
         script: |
-          echo "Structure of work folder of this pipeline:"
+
+    - task: Bash@3
+      displayName: 'Display information about this pipeline run'
+      inputs:
+        targetType: 'inline'
+        script: |
+          echo 'Using "$(dockerImagesTag)" as the tag to identify the Docker images to deploy with this pipeline run.'
+          echo
+          echo 'Structure of work folder of this pipeline:'
           tree $(Build.SourcesDirectory)
 
     - task: AzureCLI@2
       displayName: 'Install Bicep'
       inputs:
-        azureSubscription: ${{parameters.azureSubscription}}
+        azureSubscription: $(serviceConnection)
```

Comment:
Cool, is `serviceConnection` built in?

Reply:
No, this comes from a Variable Group specific to the Resource Group being deployed to.  More to be seen of this in EES-4807 PR.